src/__tests__/App.test.js

```diff
@@ -0,0 +1,14 @@
+import continentReducer, { getContinents } from '../redux/features/continentSlice';
+import { getCountries } from '../redux/features/countriesSlice';
+
+it('getContinents should return a promise', () => {
+  expect(getContinents()).toEqual(expect.any(Function));
+});
+
+it('getCountries should return a promise', () => {
+  expect(getCountries()).toEqual(expect.any(Function));
+});
+
+it('Expect continent reducer to handle getContinents', () => {
+  expect(continentReducer({}, [getContinents.fulfilled])).not.toBeNull();
+});
```

Comment:
- [ ] Please add more unit and integration tests that cover your app more in-depth.